queries/task_queries/Task_7.sql

```diff
@@ -0,0 +1,27 @@
+﻿CREATE PROCEDURE AddMoneyToCard @cardNumber nvarchar(16),
+                                @sum money
+AS
+
+BEGIN
+    BEGIN TRANSACTION
```

Comment:
Let's simplify this transaction by separating steps by variables, this code is hard to work with